atlas/responder/cmd/server/config.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import "github.com/spf13/pflag"
+
+const (
+	// configuration defaults support local development (i.e. "go run ...")
+
+	// Server
+	defaultServerAddress = "0.0.0.0"
+	defaultServerPort    = "9091"
```

Comment:
In practice, we don't change these defaults directly here; we do it in app parameters instead, for example - in deployments. This is because each our service usually has the same "defaults" as other services, and their params don't conflict with each other. Each service has its own settings, in kubernetes environment